source/MultiGrid/MultiGrid.js

```diff
@@ -644,7 +632,6 @@ export default class MultiGrid extends React.PureComponent {
         ref={this._bottomLeftGridRef}
         rowCount={Math.max(0, rowCount - fixedRowCount) + additionalRowCount}
         rowHeight={this._rowHeightBottomGrid}
-        scrollTop={scrollTop}
```

Comment:
Why this?

Reply:
Looks like it may have been an accident. Good catch.

Reply:
Oh, no never mind. I just removed the explicit `scrollTop={scrollTop}` because it wasn't necessary. I'm already spreading `{...props}` above. 😄

Reply:
The other props are explicitly passed because they have different names.